.github/workflows/insider.yaml

```diff
@@ -0,0 +1,120 @@
+name: Insider CI
+
+on:
+  schedule:
+    - cron: "0 0 * * *" # Every day at midnight
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.sha }}
+  cancel-in-progress: true
+
+jobs:
+  test:
+    strategy:
+      matrix:
+        java: ["11"]
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        version: [latest] # [x.x.x | latest | max]
+        type: [insider] # [stable | insider]
+      fail-fast: false
+
+    env:
+      CODE_VERSION: ${{ matrix.version }}
+      CODE_TYPE: ${{ matrix.type }}
+
+    runs-on: ${{ matrix.os }}
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Set up JDK 11
+        uses: actions/setup-java@v3
+        with:
+          java-version: 11
+          distribution: "temurin"
+
+      - name: Test java
+        run: java --version
+
+      - name: Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16
+          cache: npm
+
+      - name: Setup vsce
+        run: |
+          npm install -g typescript @vscode/vsce
+          vsce --version
+
+      - name: npm-ci
+        run: npm ci
+
+      - name: npm-vscode:prepublish
+        run: npm run vscode:prepublish
+
+      - name: vsce-package
+        run: vsce package
+
+      - name: test (ubuntu)
+        if: matrix.os == 'ubuntu-latest'
+        run: xvfb-run --auto-servernum npm test
+
+      - name: test (macos, windows)
+        if: matrix.os != 'ubuntu-latest'
+        run: npm test
+
+      - name: ui test (ubuntu)
+        id: uiTest_Ubuntu
+        if: matrix.os == 'ubuntu-latest'
+        run: xvfb-run --auto-servernum npm run ui-test
+
+      # - name: ui test (macos, windows)
+      #   id: uiTest_MacOS_Windows
+      #   if: matrix.os != 'ubuntu-latest'
+      #   run: npm run ui-test
```

Comment:
Remind to have a jira ticket to deal with UI tests on MacOS and Windows. Providing the ticket number in the comment would be nice.

Reply:
> * please create an issue to test with the minimal version of the API which was previously handled with the vscode-versions-matrix.yaml

Did I understand correctly that I need to add a ["1.67.2"] version to the main matrix? :thinking:

Reply:
> Remind to have a jira ticket to deal with UI tests on MacOS and Windows. Providing the ticket number in the comment would be nice.

> Remind to have a jira ticket to deal with UI tests on MacOS and Windows. Providing the ticket number in the comment would be nice.

[FUSETOOLS2-1998](https://issues.redhat.com/browse/FUSETOOLS2-1998)

Reply:
Done with + ["1.67.2"]